tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -1964,7 +1964,7 @@ declare namespace ts {
         DiagnosticsPresent_OutputsSkipped = 1,
         DiagnosticsPresent_OutputsGenerated = 2,
         InvalidProject_OutputsSkipped = 3,
-        ProjectReferenceCycle_OutputsSkupped = 4
```

Comment:
Need to mark the incorrect one as `@deperecated` instead of removing since its api change.

Reply:
Done.